Modules/CustomRolesHelper.cs

```diff
@@ -1689,6 +1690,18 @@ public static bool CheckAddonConfilct(CustomRoles role, PlayerControl pc)
                 if ((pc.GetCustomRole().IsCrewmate() && !Silent.CanBeOnCrew.GetBool()) || (pc.GetCustomRole().IsNeutral() && !Silent.CanBeOnNeutral.GetBool()) || (pc.GetCustomRole().IsImpostor() && !Silent.CanBeOnImp.GetBool()))
                     return false;
                 break;
+            case CustomRoles.Susceptible:
+                if ((pc.GetCustomRole().IsCrewmate() && !Susceptible.CanBeOnCrew.GetBool()) || (pc.GetCustomRole().IsNeutral() && !Susceptible.CanBeOnNeutral.GetBool()) || (pc.GetCustomRole().IsImpostor() && !Susceptible.CanBeOnImp.GetBool()))
+                return false;
+                break;
+            case CustomRoles.Tired:
+                if (pc.Is(CustomRoles.Farseer)
+               || pc.Is(CustomRoles.Alchemist)
+               || pc.Is(CustomRoles.Torch)
+                    ) return false;
```

Comment:
basically all roles/add-ons that has vision, idk if I missed some?

Reply:
Bewilder
Lighter

Reply:
Oh shit, right, but I think rn it's only flash and alchemist tough. well this should be all I think?

Reply:
Mare i think